language/move-lang/src/parser/syntax.rs

```diff
@@ -1315,13 +1358,8 @@ fn parse_function_decl<'input>(
         None
     };
 
-    // (<Public>)?
-    let public_opt = consume_optional_token_with_loc(tokens, Tok::Public)?;
-    let visibility = if let Some(loc) = public_opt {
-        FunctionVisibility::Public(loc)
-    } else {
-        FunctionVisibility::Internal
-    };
+    // <FunctionVisibility>
```

Comment:
I don't think you need the comment here. It should be clear from the call what this is parsing.